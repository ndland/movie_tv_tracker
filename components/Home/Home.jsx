import Header from '../Header/Header';

const Home = () => <Header />;

export default Home;
