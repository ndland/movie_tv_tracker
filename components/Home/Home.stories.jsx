import Home from './Home';
import { SessionProvider } from 'next-auth/react';

export default {
  title: 'Components/Home',
  component: Home,
};

const session = {
  user: {
    email: 'example@example.com',
    image:
      'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRIo9zw5uLmmB_BUiCiNwbrb_mnR7HdYqq-nw&usqp=CAU',
  },
};

export const Default = () => (
  <SessionProvider session={session}>
    <Home />
  </SessionProvider>
);
