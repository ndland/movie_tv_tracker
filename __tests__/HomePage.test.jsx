import React from 'react';
import { render, screen } from '@testing-library/react';
import Home from '../pages/index';

describe('Home Page', () => {
  const defaultProps = {};
  const { container } = render(<Home {...defaultProps} />);

  it('renders', () => {
    expect(container).toMatchSnapshot();
  });
});
