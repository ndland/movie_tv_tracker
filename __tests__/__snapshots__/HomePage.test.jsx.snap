// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders 1`] = `
<div>
  <div
    class="flex flex-col p-2"
  >
    <div
      class="flex flex-col -space-y-px shadow-sm"
    >
      <h2
        class="text-3xl font-extrabold mb-4 self-center text-slate-600"
      >
        Login
      </h2>
      <label
        class="sr-only"
        for="username"
      >
        Username
      </label>
      <input
        aria-label="username"
        class="border p-2 rounded-t-md"
        placeholder="Username"
        type="text"
      />
      <label
        class="sr-only"
        for="password"
      >
        Password
      </label>
      <input
        aria-label="password"
        class="border p-2 rounded-b-md"
        placeholder="Password"
        type="password"
      />
    </div>
    <button
      class="text-white text-xl font-bold bg-slate-600 p-2 rounded-md mt-4"
    >
      Login
    </button>
  </div>
</div>
`;
